src/layers/HTMLLayer.ts

```diff
@@ -0,0 +1,63 @@
+import { select, Selection } from 'd3-selection';
+import { Layer } from './Layer';
+import { OnMountEvent, OnResizeEvent } from '../interfaces';
+
+const DEFAULTWIDTH = 200;
+const DEFUALTHEIGHT = 300;
```

Comment:
Default sizes for the layers should be in one common location to ensure that different layers do not end up with different sizes if the user does not provide size. Not a big deal tho, we can fix this later.